From 7c7a7a356d4e74722292d2bf69e7fd1513d6d679 Mon Sep 17 00:00:00 2001
From: Summer Qin <summer.qin@arm.com>
Date: Tue, 29 Aug 2023 17:08:19 +0800
Subject: [PATCH] Place crypto test 16 at the beginning of the test

HMAC_DRBG random generator is slower than CTR_DRBG, and this
will influence the time spending for test 16 RSA 2048. Placing
test 16 at the beginning will be faster and mitigate this issue.

Signed-off-by: Summer Qin <summer.qin@arm.com>
---
 api-tests/dev_apis/crypto/testsuite.db | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/api-tests/dev_apis/crypto/testsuite.db b/api-tests/dev_apis/crypto/testsuite.db
index 47ee724..bf6bffa 100644
--- a/api-tests/dev_apis/crypto/testsuite.db
+++ b/api-tests/dev_apis/crypto/testsuite.db
@@ -20,6 +20,7 @@
 
 (START)
 
+test_c016
 test_c001
 test_c002
 test_c003
@@ -35,7 +36,6 @@ test_c012
 test_c013
 test_c014
 test_c015
-test_c016
 test_c017
 test_c018
 test_c019
-- 
2.25.1

